charts/mlbench/templates/worker-statefulset.yaml

```diff
@@ -1,3 +1,30 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "mlbench.worker.fullname" . }}-sa
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "mlbench.worker.fullname" . }}-role
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
+---
+# This cluster role binding allows anyone in the "manager" group to read secrets in any namespace.
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "mlbench.worker.fullname" . }}-rolebinding
+roleRef:
+  kind: Role
+  name: {{ template "mlbench.worker.fullname" . }}-role
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: {{ template "mlbench.worker.fullname" . }}-sa
+---
```

Comment:
Each helm resource should be in a separate file, if we want to release that chart to the Helm Chart repository at some point.

See https://github.com/helm/helm/blob/master/docs/chart_best_practices/templates.md

Reply:
OK. Thanks.